.github/workflows/release.yml

```
@@ -68,21 +87,21 @@ jobs:
-            ls -lh "Git-${{ steps.version.outputs.version }}-unsigned.modl"
+            cp "$MODL_FILE" "Git-${{ steps.version.outputs.tag_version }}-unsigned.modl"
+            echo "Preserved unsigned module: Git-${{ steps.version.outputs.tag_version }}-unsigned.modl"
+            ls -lh "Git-${{ steps.version.outputs.tag_version }}-unsigned.modl"
           else
             echo "Error: No .modl file found"
             exit 1
           fi
 
       - name: Decode keystore
-        if: steps.version.outputs.is_release == 'true' && secrets.KEYSTORE_BASE64 != ''
+        if: secrets.KEYSTORE_BASE64 != ''
```

actionlint (1.7.10)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)